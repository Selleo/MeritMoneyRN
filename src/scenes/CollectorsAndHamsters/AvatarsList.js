import React, { Component } from 'react'
import { View } from 'react-native'

import Avatar from 'src/components/Avatar'
export default class AvatarsList extends Component {
  render() {
    return (
      <View>
        <Avatar size={100} />
        <Avatar size={100} />
        <Avatar size={100} />
        <Avatar size={100} />
        <Avatar size={100} />
        <Avatar size={100} />
      </View>
    )
  }
}
