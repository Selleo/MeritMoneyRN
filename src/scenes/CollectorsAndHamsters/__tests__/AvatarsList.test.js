import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'

import AvatarsList from '../AvatarsList'

describe('<AvatarsList />', () => {
  const wrapper = renderer.create(<AvatarsList />)

  test('render', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
