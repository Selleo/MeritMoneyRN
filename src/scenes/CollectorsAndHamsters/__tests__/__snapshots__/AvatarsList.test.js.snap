// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AvatarsList /> render 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "flexWrap": "wrap",
      "justifyContent": "space-between",
      "paddingHorizontal": 20,
    }
  }
>
  <View
    style={
      Object {
        "paddingHorizontal": 15,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#ffffff",
          },
          Object {
            "borderRadius": 40,
            "height": 80,
            "width": 80,
          },
        ]
      }
    >
      <Image
        source={
          Object {
            "uri": "https://i.imgur.com/ckqCXMS.png",
          }
        }
        style={
          Array [
            Object {
              "borderRadius": 40,
              "height": 80,
              "width": 80,
            },
            Object {
              "zIndex": -1,
            },
          ]
        }
      />
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#ffffff",
          "paddingVertical": 5,
          "textAlign": "center",
        }
      }
    >
      Dominik
      

       Duda
    </Text>
  </View>
  <View
    style={
      Object {
        "paddingHorizontal": 15,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#ffffff",
          },
          Object {
            "borderRadius": 40,
            "height": 80,
            "width": 80,
          },
        ]
      }
    >
      <Image
        source={
          Object {
            "uri": "https://i.imgur.com/ckqCXMS.png",
          }
        }
        style={
          Array [
            Object {
              "borderRadius": 40,
              "height": 80,
              "width": 80,
            },
            Object {
              "zIndex": -1,
            },
          ]
        }
      />
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#ffffff",
          "paddingVertical": 5,
          "textAlign": "center",
        }
      }
    >
      Jak
      

      Smiesz
    </Text>
  </View>
  <View
    style={
      Object {
        "paddingHorizontal": 15,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#ffffff",
          },
          Object {
            "borderRadius": 40,
            "height": 80,
            "width": 80,
          },
        ]
      }
    >
      <Image
        source={
          Object {
            "uri": "https://i.imgur.com/ckqCXMS.png",
          }
        }
        style={
          Array [
            Object {
              "borderRadius": 40,
              "height": 80,
              "width": 80,
            },
            Object {
              "zIndex": -1,
            },
          ]
        }
      />
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#ffffff",
          "paddingVertical": 5,
          "textAlign": "center",
        }
      }
    >
      Bartłomiej
      

      Wójtowicz
    </Text>
  </View>
  <View
    style={
      Object {
        "paddingHorizontal": 15,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#ffffff",
          },
          Object {
            "borderRadius": 40,
            "height": 80,
            "width": 80,
          },
        ]
      }
    >
      <Image
        source={
          Object {
            "uri": "https://i.imgur.com/ckqCXMS.png",
          }
        }
        style={
          Array [
            Object {
              "borderRadius": 40,
              "height": 80,
              "width": 80,
            },
            Object {
              "zIndex": -1,
            },
          ]
        }
      />
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#ffffff",
          "paddingVertical": 5,
          "textAlign": "center",
        }
      }
    >
      Grzegorz
      

      aaaaa
    </Text>
  </View>
  <View
    style={
      Object {
        "paddingHorizontal": 15,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#ffffff",
          },
          Object {
            "borderRadius": 40,
            "height": 80,
            "width": 80,
          },
        ]
      }
    >
      <Image
        source={
          Object {
            "uri": "https://i.imgur.com/ckqCXMS.png",
          }
        }
        style={
          Array [
            Object {
              "borderRadius": 40,
              "height": 80,
              "width": 80,
            },
            Object {
              "zIndex": -1,
            },
          ]
        }
      />
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#ffffff",
          "paddingVertical": 5,
          "textAlign": "center",
        }
      }
    >
      Panie
      

      Janie
    </Text>
  </View>
  <View
    style={
      Object {
        "paddingHorizontal": 15,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#ffffff",
          },
          Object {
            "borderRadius": 40,
            "height": 80,
            "width": 80,
          },
        ]
      }
    >
      <Image
        source={
          Object {
            "uri": "https://i.imgur.com/ckqCXMS.png",
          }
        }
        style={
          Array [
            Object {
              "borderRadius": 40,
              "height": 80,
              "width": 80,
            },
            Object {
              "zIndex": -1,
            },
          ]
        }
      />
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#ffffff",
          "paddingVertical": 5,
          "textAlign": "center",
        }
      }
    >
      Rano
      

      Wstań
    </Text>
  </View>
</View>
`;
